src/Razor/src/Microsoft.AspNetCore.Razor.Language/RazorParserOptions.cs

```diff
@@ -15,7 +15,7 @@ public static RazorParserOptions CreateDefault()
                 designTime: false,
                 parseLeadingDirectives: false,
                 version: RazorLanguageVersion.Latest,
-                fileKind: null);
+                fileKind: FileKinds.Legacy);
```

Comment:
Is the idea that `RazorParserOptions.FileKind` will always be non-null? If so, there are a few more places that need to change.

Reply:
hmm - I thought I got them all. I'll take another look.

Reply:
There is one place in `CreateDesignTime()` in this file. And also can probably remove this condition here https://github.com/aspnet/AspNetCore-Tooling/blob/master/src/Razor/src/Microsoft.AspNetCore.Razor.Language/DefaultRazorDirectiveFeature.cs#L41.

I guess it makes sense to add an arg null check in the DefaultRazorParserOptions ctor.

Reply:
I review this, I was only missing one case for tests.

Reply:
What about line 23 and line 42 in this file? It passes `fileKind: null`. Is that okay? I'd rather us either always pass in `FileKinds.Legacy` by default or just pass `null` everywhere we don't have it and let the consuming code handle fileKind being null. Just want it to be consistent.

Reply:
I changed this since you asked. 

It doesn't really matter becuase those places just *initialize* the builder. The builder will always provide a non-null value when you call build.